static/app/components/feedback/useFetchFeedbackIssue.tsx

```diff
@@ -38,6 +38,7 @@ export default function useFetchFeedbackIssue(
 
   return {
     issueData: issueData ? hydrateFeedbackRecord(issueData) : undefined,
```

Comment:
originally i was passing in `eventData` to `hydrateFeedbackRecord` but this is incompatible with the mapping in `useFetchFeedbackInfiniteListData`